src/test/java/redis/clients/jedis/tests/commands/SortedSetCommandsTest.java

```diff
@@ -542,7 +545,147 @@ public void zscore() {
     assertNull(bscore);
 
   }
+  
+  @Test
+  public void zpopmax() {
+    jedis.zadd("foo", 1d, "a");
+    jedis.zadd("foo", 10d, "b");
+    jedis.zadd("foo", 0.1d, "c");
+    jedis.zadd("foo", 2d, "d");
+
+    Tuple wrapper = jedis.zpopmax("foo");
+    String member = wrapper.getElement();
+    Double score = wrapper.getScore();
+    
+    assertEquals("b", member);
+    assertEquals((Double) 10d, score);
+
+    wrapper = jedis.zpopmax("foo");
+    member = wrapper.getElement();
+    score = wrapper.getScore();
+    
+    assertEquals("d", member);
+    assertEquals((Double) 2d, score);
+
+    wrapper = jedis.zpopmax("foo");
+    member = wrapper.getElement();
+    score = wrapper.getScore();
+    
+    assertEquals("a", member);
+    assertEquals((Double) 1d, score);
+    
+    wrapper = jedis.zpopmax("foo");
+    member = wrapper.getElement();
+    score = wrapper.getScore();
+    
+    assertEquals("c", member);
+    assertEquals((Double) 0.1d, score);
+    
+    // Empty
+    wrapper = jedis.zpopmax("foo");
+    assertTrue(wrapper == null);
```

Comment:
Please keep one coding pattern for binary and non-binary within same method. Personally, I liked your code for binary better. E.g.
1. `actual`, `expected` variables
2. `assertNull(actual)` instead of `assertTrue(wrapper == null)`

Reply:
In the spirit of using the same thing across multiple places, I hope this is OK? 
```
    expected = null;
    assertEquals(expected, actual);
```

Instead of `assertNull(actual)`.

Reply:
At least I would prefer that over `assertTrue(wrapper == null)`.